### ISSUE TITLE
Searches on image produced `AttributeError` on `license_url` in production

### ISSUE THREAD

**Issue by AetherUnbound** [WordPress/openverse-api#704]
## Description

A recent deployment of [v2.5.1](https://github.com/WordPress/openverse-api/releases/tag/v2.5.1) to production resulted in failures for all image searches. This included `image/` searches and searches for related images. We were _not_ able to replicate this in staging, and https://api-dev.openverse.engineering is currently running v2.5.1 without issue.

```
KeyError: 'license_url'
  File "elasticsearch_dsl/utils.py", line 159, in __getattr__
    return self.__getitem__(attr_name)
  File "elasticsearch_dsl/utils.py", line 178, in __getitem__
    return _wrap(self._d_[key])

AttributeError: 'Hit' object has no attribute 'license_url'
  File "rest_framework/fields.py", line 457, in get_attribute
    return get_attribute(instance, self.source_attrs)
  File "rest_framework/fields.py", line 97, in get_attribute
    instance = getattr(instance, attr)
  File "elasticsearch_dsl/utils.py", line 161, in __getattr__
    raise AttributeError(

AttributeError: Got AttributeError when attempting to get a value for field `license_url` on serializer `ImageSerializer`.
The serializer field might be named incorrectly and not match any attribute or key on the `Hit` instance.
Original exception text was: 'Hit' object has no attribute 'license_url'.
  File "django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
  File "django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "rest_framework/viewsets.py", line 125, in view
    return self.dispatch(request, *args, **kwargs)
  File "rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
  File "rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
  File "django/utils/decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "catalog/api/views/media_views.py", line 84, in list
    return self.get_paginated_response(serializer.data)
  File "rest_framework/serializers.py", line 768, in data
    ret = super().data
  File "rest_framework/serializers.py", line 253, in data
    self._data = self.to_representation(self.instance)
  File "rest_framework/serializers.py", line 686, in to_representation
    return [
  File "rest_framework/serializers.py", line 687, in <listcomp>
    self.child.to_representation(item) for item in iterable
  File "catalog/api/serializers/media_serializers.py", line 378, in to_representation
    repr = super().to_representation(*args, **kwargs)
  File "rest_framework/serializers.py", line 509, in to_representation
    attribute = field.get_attribute(instance)
  File "rest_framework/fields.py", line 490, in get_attribute
    raise type(exc)(msg)
```

## Additional context

**Sentry alert**: https://sentry.io/share/issue/06f2197af79f4e4e942668fcc3a94eae/

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

**Comment by dhruvkb** [comment]
This should be fixed by #696. @AetherUnbound can you try to reproduce this on `main` again?

**Comment by AetherUnbound** [comment]
I can confirm this is resolved (tested with commit 0782bce275e91aa95d3b533a547a4ec3e9763ca1)! Here were the steps I took to reproduce:
1. Remove the `meta_data` field from the first row of `sample_data/sample_images.csv`
2. `just recreate && just init`
3. Visit http://localhost:8000/v1/images/?q=dress (the first record is returned when searching for "dress")

On v2.5.1 this will fail with an `AttributeError` as experienced in production. As of the aforementioned commit, the results get returned (and the first entry has a `null` `license_url` field).
